apps/mobile/src/components/Post/PostInput.tsx

```diff
@@ -34,67 +38,72 @@ export function PostInput({ value, onChange, tokenRef, mentions, onSelectionChan
-        height: 117,
-      }}
-    >
-      <MentionTextInput
-        value={value}
-        onChangeText={onChange}
-        className="px-3 pt-3 text-sm"
-        selectionColor={colorScheme === 'dark' ? colors.white : colors.black['800']}
-        placeholderTextColor={colorScheme === 'dark' ? colors.metal : colors.shadow}
-        multiline
-        onSelectionChange={(e: NativeSyntheticEvent<TextInputSelectionChangeEventData>) => {
-          onSelectionChange(e.nativeEvent.selection);
-        }}
-        maxLength={MAX_LENGTH}
-        autoCapitalize="none"
-        autoComplete="off"
-        keyboardAppearance={colorScheme}
-        placeholder={inputPlaceHolder}
+    <View className="space-y-2">
+      <View
+        className="relative border bg-faint dark:bg-black-900 border-porcelain dark:border-black-500"
         style={{
-          flex: 1,
-          color: colorScheme === 'dark' ? colors.white : colors.black['800'],
+          minHeight: 117,
+          height: 'auto',
         }}
-        mentions={mentions}
-      />
-
-      {isTextTooLong && (
-        <Typography
-          className="text-sm text-red absolute bottom-2 left-2"
-          font={{
-            family: 'ABCDiatype',
-            weight: 'Medium',
+      >
+        <MentionTextInput
+          value={value}
+          onChangeText={onChange}
+          className="p-3 text-sm"
+          selectionColor={colorScheme === 'dark' ? colors.white : colors.black['800']}
+          placeholderTextColor={colorScheme === 'dark' ? colors.metal : colors.shadow}
+          multiline
+          onSelectionChange={(e: NativeSyntheticEvent<TextInputSelectionChangeEventData>) => {
+            onSelectionChange(e.nativeEvent.selection);
           }}
-        >
-          Your text is too long
-        </Typography>
-      )}
-      {value.length > 0 && (
-        <Typography
-          className={clsx(
-            'text-sm absolute bottom-3 right-3',
-            isTextTooLong ? 'text-red' : 'text-metal'
-          )}
-          font={{
-            family: 'ABCDiatype',
-            weight: isTextTooLong ? 'Medium' : 'Regular',
+          maxLength={MAX_POST_LENGTH}
+          autoCapitalize="none"
+          autoComplete="off"
+          keyboardAppearance={colorScheme}
+          placeholder={inputPlaceHolder}
+          style={{
+            color: colorScheme === 'dark' ? colors.white : colors.black['800'],
           }}
-        >
-          {characterCount}/600
-        </Typography>
-      )}
+          mentions={mentions}
+        />
+      </View>
+      <View className="flex-row items-center justify-between">
+        {isTextTooLong ? (
+          <Animated.View entering={transitionFadeIn} exiting={transitionFadeOut}>
+            <Typography
+              className="text-sm text-red"
+              font={{
+                family: 'ABCDiatype',
+                weight: 'Medium',
+              }}
+            >
+              Max text level reached
```

Comment:
length*, not level :)

you've been playing too much pokemon

Reply:
lol 😂. Updated it!